jbosstools/configuration/ide-config.properties

```diff
@@ -8,7 +8,7 @@ jboss.central.webpage.url|jbosstools|4.15.0.AM1=http://download.jboss.org/jbosst
 jboss.discovery.site.integration-stack.url|jbosstools|4.15.0.AM1=http://download.jboss.org/jbosstools/photon/stable/updates/integration-stack/
```

Comment:
Why stable ? No snapshots for IS ?

Reply:
No, it has been like this since a while now..